src/ledger-bridge-keyring.test.ts

```diff
@@ -0,0 +1,845 @@
+import { strict as assert } from 'assert';
+import chai from 'chai';
+import spies from 'chai-spies';
+import EthereumTx from 'ethereumjs-tx';
+import HDKey from 'hdkey';
+import * as ethUtil from 'ethereumjs-util';
+import { TransactionFactory } from '@ethereumjs/tx';
+import Common from '@ethereumjs/common';
+import sigUtil from 'eth-sig-util';
+import windowShim from '../test/window.shim';
+import documentShim from '../test/document.shim';
+
+import { AccountDetails, LedgerBridgeKeyring } from './ledger-bridge-keyring';
+
+global.document = documentShim;
+global.window = windowShim;
+
+const { expect } = chai;
```

Comment:
Interesting. Is it worth changing the chai import above to `import chai, { expect } from 'chai'`?